apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/PublisherClient.cs

```diff
@@ -327,10 +382,12 @@ public sealed class PublisherClientImpl : PublisherClient
         private class Batch
         {
             public TaskCompletionSource<IList<string>> BatchCompletion { get; } = new TaskCompletionSource<IList<string>>();
-            public List<PubsubMessage> Messages { get; } = new List<PubsubMessage>();
-            public CancellationTokenSource TimerCts { get; } = new CancellationTokenSource();
+            public List<PubsubMessage> Messages { get; private set; } = new List<PubsubMessage>();
+            public CancellationTokenSource ProcessedCts { get; } = new CancellationTokenSource();
```

Comment:
What does "processed" mean here? I've seen it multiple times, and it's unclear to me.